README.md

```diff
@@ -1,46 +1,103 @@
-# SecretHub Cross Language Golang Client<sup>**experimental**</sup>
+# SecretHub-XGO Guide for C# integration
```

Comment:
I think this is still as if it's a guide, not the readme.

Reply:
Adjusted. Let me know if it is ok.